modules/web/src/com/haulmont/addon/admintools/web/tomcat/TomcatWebMBean.java

```diff
@@ -0,0 +1,29 @@
+package com.haulmont.addon.admintools.web.tomcat;
+
+import org.springframework.jmx.export.annotation.ManagedOperation;
+import org.springframework.jmx.export.annotation.ManagedOperationParameter;
+import org.springframework.jmx.export.annotation.ManagedOperationParameters;
+import org.springframework.jmx.export.annotation.ManagedResource;
+
+import java.io.IOException;
+
+@ManagedResource(description = "Operations for Tomcat container on a web module")
```

Comment:
Operations on Tomcat running the web client block

Reply:
fixed